test/jdk/jdk/crac/recursiveCheckpoint/Test.java

```diff
@@ -26,6 +26,7 @@
 /*
  * @test
  * @summary check that the recursive checkpoint is not allowed
+ * @requires (os.family == "linux")
```

Comment:
I believe this test does not need `pauseengine` at all and there even may be errors because of it, but I am going to test this internally first